src/main/java/io/numaproj/numaflow/batchmapper/Server.java

```diff
@@ -0,0 +1,128 @@
+package io.numaproj.numaflow.batchmapper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+import io.grpc.ServerBuilder;
+import io.numaproj.numaflow.info.ServerInfoAccessor;
+import io.numaproj.numaflow.info.ServerInfoAccessorImpl;
+import io.numaproj.numaflow.shared.GrpcServerUtils;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * MapServer is the gRPC server for executing map operation.
```

Comment:
```suggestion
 * Server is the gRPC server for executing batch map operation.
```

Please also update it for mapstreamer.

Reply:
Done